tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,352 @@
+    user clicks radio   Ready for higher review
+    user enters text into element  id:releaseStatusForm-internalReleaseNote     ready for higher review (publication owner)
+    user clicks button  Update status 
+    user waits until element is visible  //*[@id="CurrentReleaseStatus-Awaiting higher review"]
+    
+Assert publication owner can edit release status to "In draft"
+    [Tags]  HappyPath
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status  30
+    user clicks radio   In draft
+    user enters text into element  id:releaseStatusForm-internalReleaseNote     Moving back to Draft state (publication owner)
+    user clicks button  Update status 
+
+User goes back to admin dashboard
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+
+Assert that a publication owner can make a new release 
+    [Tags]  HappyPath
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+    user opens accordion section  ${PUBLICATION_NAME}
+    user waits until page does not contain loading spinner
+    user clicks link  Create new release
+    user creates release for publication  ${PUBLICATION_NAME}  Academic Year Q1  2020
+
+Assert publication owner can upload subject file on new release
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user uploads subject   ${SUBJECT_NAME}  seven_filters.csv  seven_filters.meta.csv
+    
+Assert publication owner can add meta guidance to ${SUBJECT_NAME} on new release
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until page does not contain loading spinner
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user enters text into meta guidance data file content editor  ${SUBJECT_NAME}
+    ...  meta guidance content
+    user clicks button  Save guidance
+  
+Go to "Sign off" page as publication owner 
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until h2 is visible  Sign off
+     
+Assert publication owner cannot approve release for immediate publication on new release
+    [Tags]  HappyPath
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status  60
+    user scrolls to element  //*[@id="releaseStatusForm-approvalStatus-Approved"]
+    user checks element is disabled  //*[@id="releaseStatusForm-approvalStatus-Approved"]
+
+Navigate to administration as bau1 to remove publication owner access
+    [Tags]  HappyPath
+    user changes to bau1
+    user goes to url  %{ADMIN_URL}/administration/users
+
+Navigate to manage users
+    [Tags]  HappyPath    
+    user clicks link  Manage  xpath://td[text()="ees-analyst1@education.gov.uk"]/..
+    user waits until page does not contain loading spinner
+
+Remove publication owner access 
+    [Tags]  HappyPath
+    user waits for page to finish loading    
```

Comment:
I think the next line is all that is required to ensure we wait, so this can be removed.

This keyword is nothing but a `sleep 0.2`. In most cases it is better to wait on a element than use this. This keyword was created as something to turn to as a last resort